hub/examples/image_retraining/label_image.py

```diff
@@ -41,67 +42,26 @@
 
 def wiki(celestial_object):
     ans = celestial_object
+    cwd = os.getcwd()
+    with open(os.path.join(cwd, 'hub', 'examples', 'image_retraining', 'display_info.yml'), 'r') as stream:
```

Comment:
@Indus5 It should be `with open(os.path.join(cwd, 'display_info.yaml'), 'r') as stream:`